svnwiki-mode.el

```diff
@@ -1,11 +1,13 @@
-;;; svnwiki-mode.el --- Major mode for editing svnwiki markup
+;;; svnwiki-mode.el --- Major mode for editing svnwiki markup -*- lexical-binding: t -*-
 
 ;; Copyright (C) 2016 Vasilij Schneidermann <mail@vasilij.de>
+;; SPDX-License-Identifier: GPL-3.0-or-later
 
 ;; Author: Vasilij Schneidermann <mail@vasilij.de>
-;; URL: https://depp.brause.cc/svnwiki-mode
+;; URL: https://github.com/chicken-contrib/svnwiki-mode
 ;; Version: 0.0.1
-;; Keywords: text
+;; Package-Requires: ((emacs "24.1"))
+;; Keywords: wp, hypermedia, languages
```

Comment:
What does wp mean? Word processing? Why is languages relevant, but text isn't?

Reply:
Yeah, word processing. package-lint says the keywords are supposed to come from the set `finder-known-keywords`. That set doesn't contain `text` for some reason even though there exists a `text` customize group. I copied the above list of keywords from an Emacs mode for some popular markup language, can't remember which one.

Reply:
I've run package-lint and apparently its complaint is that not a single standardized keyword from `finder-known-keywords` is used. Adding one such keyword silences the warning. There is no need to only use standardized keywords. Looking through that list I'm not sure any of these is a good match:

- hypermedia: "links between text or other media type" <- This mode doesn't do anything special with links, other than highlighting them. goto-address-mode though does.
- languages: "specialized modes for editing programming language" <- Not a programming language.
- wp: "word processing" <- This is the closest match, but for no obvious reason. I'd rather consider features such as enriched text or smart quotes word processing. It's strange how there's a keyword for programming languages, but not for text modes (although TeX is included, lol).

Reply:
What are the keywords used for anyway? The MELPA maintainers complain that everyone uses the keywords in a chaotic manner with little rhyme or reason.

Reply:
It's possible to view packages by keywords. This would require the Emacs team to extend the list of keywords though as packages keep exceeding the initial scope covered by the keywords.

Reply:
Sounds like a few maintainers should sort through the entire package collection and come up with some keywords over a weekend sometime, which they are unlikely to want to do :) I'm fine with any set of keywords package-lint is happy with.